File: Server/Entities/Service.cs
namespace API.Entities
{
    public class Service
    {
        public int Id { get; set; }
        public int OfficeId { get; set; }
        public string OfficeName { get; set; }
        public string Name { get; set; }
        public string Unit { get; set; }
        public decimal Price { get; set; }
        public decimal Vat { get; set; }
        public int Qty { get; set; }
        public decimal Discount { get; set; }
    }
}

File: client/src/schema/service.js
/**
 * serviceSchema is an array of objects, each describing a property field.
 *  Each object contains:
 *   - field: string (the property key)
 *   - header: string (the display name)
 *   - type: the data type, e.g., "string" or "decimal"

 * @type {Array<{field: string, header: string, type: string}>}
 */
const serviceSchema = [
    { field: "officeName", header: "Kohde", type: "string" },
    { field: "name", header: "Nimi", type: "string" },
    { field: "unit", header: "Yksikkö", type: "string" },
    { field: "price", header: "Hinta", type: "decimal" },
    { field: "vat", header: "ALV %", type: "decimal" },
]

export default serviceSchema;